great-idea-website/css/index.css

```diff
@@ -63,4 +63,122 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+hr {
+	width: 80%;
+}
+.main-content {
+	margin: 0 20%;
+}
+
+nav {
+	width: 70%;
+	display: inline-block;
+	margin: 0 0 0 5%;
+
+}
+nav a {
+	text-decoration: none;
+	display: inline-block;
+	margin: 5%;
+}
+
+.logo {
+	display: inline-block;
+}
+
+.top-feature {
+	margin: 5% 10%;
+}
+
+.feature-text {
+	display: inline-block;
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+}
+
+.first-h2 {
+	text-align: center;
+}
+
+.second-h2 {
+	text-align: center;
+}
+
+.third-h2 {
+	text-align: center;
+}
+
+.get-started-btn {
+	margin-left: 25%;
+	width: 50%;
+	height: 15%;
+	background-color: #fff;
+}
+
+.feature-pic {
+	display: inline-block;
+	Vertical-align: top;
+}
+
+.top-content {
+	margin: 5% 10%;
+	padding-left: 2%;
+}
+
+.features {
+	display: inline-block;
+	width: 40%;
+	margin: 2% 5%;
+}
+
+.about {
+	display: inline-block;
+	width: 40%;
+}
+
+.middle-img {
+	margin: 0 6%;
```

Comment:
I'll have to double-check on this, but I think the danger of using percentages for margins is that it may not be consistent, as percentages are based off the size of the parent (_?_) container. This ultimately means "6%" margin might be different for different elements in different parts of the page. Not great if you're trying to have consistent margins.